content/docs/cucumber/api.md

```diff
@@ -23,7 +23,23 @@ The number of parameters in the {{% stepdef-body %}} has to match the number of
 
 ## Data Tables
 
-The simplest way to pass a {{% text "java,kotlin" %}}`List<String>`{{% /text %}}{{% text "scala" %}}`java.util.List[String]`{{% /text %}} to a step definition is to use a data table:
+Data tables from Gherkin can be accessed by using the DataTable object as the last parameter in a step definition.
+This conversion can be done either by Cucumber or manually.
+
+{{% text "java,kotlin" %}}Depending on the table shape as one of the following collections:{{% text "java,kotlin" %}}
```

Comment:
This {{ seems unpaired - the opening and closing {{'s do not line up, do they?

Reply:
Looks perfectly balanced to me.

Reply:
You are correct. Thanks for catching this.

Reply:
Should be fixed now